thesiscentral-vireo/dataspace/python/IMPORT.md

```diff
@@ -0,0 +1,278 @@
+[Google Drive](https://drive.google.com/file/d/1yVsV5PG-WPtj-eV7lHGRbuj3sVUGdwZh/view?usp=sharing) to `downloads/Restrictions.xlsx`).
+
+Then, one must copy the file, and add an `ID` column to this exported Spreadsheet:
+```
+cp downloads/Restrictions.xlsx downloads/RestrictionsWithId.xlsx
+```
+
+![alt text](./docs/thesis-central_screenshot_1.png)
+![alt text](./docs/thesis-central_screenshot_2.png)
+
+Finally, one must update the spreadsheet with the following:
+
+```bash
+/bin/tcsh
+set department="Mechanical & Aerospace Engr"
+
+pipenv install lxml pandas
+cp -r "downloads/$department" "export/"
+# This shouldn't be necessary, but requires that the Python scripts be rewritten
+cp downloads/RestrictionsWithId.xlsx export/RestrictionsWithId.xlsx
+
+cp downloads/RestrictionsWithId.xlsx "export/$department/RestrictionsWithId.xlsx"
+
+python restrictionsFindIds.py --thesis "export/$department/ExcelExport.xlsx" --restrictions "export/$department/RestrictionsWithId.xlsx"
+
+cp ImportRestrictions.xlsx "export/$department/RestrictionsWithId.xlsx"
+```
+
+## Adding the Academic Programs
+
+Academic programs are listed in a spreadsheet located on [Google
+Drive](https://drive.google.com/file/d/1K_rrBPY-Pf3DcqbCS-ZxYFjMQl3bIYEM/view?usp=sharing).
+
+This should please be downloaded and copied with the following:
+```
+cp export/AdditionalPrograms.xlsx "export/$department/AdditionalPrograms.xlsx"
+```
+
+## Building DSpace Submission Information Packages (SIPs)
+
+Please note that this assumes that you have downloaded the Thesis Central 
+departmental Excel Spreadsheet into `downloads/ExcelExport.xlsx`, and
+the departmental DSpace Simple Archive into `downloads/DSpaceSimpleArchive.zip`.
+
+### Cleaning the Environment
+
+```bash
+/bin/tcsh
+set department="English"
+source clean-simple-archives
+```
+
+### Initializing the Environment
+
+```bash
+/bin/tcsh
+set department="English"
+
+cd "export/$department"
+unzip DSpaceSimpleArchive.zip
+cd -
+```
+
+### Building SIPs
+
+```bash
+/bin/tcsh
+set department="English"
+pipenv install lxml pandas
+
+# Please note that the DSpaceSimpleArchive is decompressed
+cd "export/$department"
+rm -r ./DSpaceSimpleArchive
+unzip DSpaceSimpleArchive.zip
+cd -
+
+source prepare-to-dataspace "export/$department"
+```
+
+#### Rebuilding SIPs
+
+```
+rm "export/$department/enhanceAips.trace"
+rm "export/$department/$department.tgz"
+rm -fr "export/$department/DSpaceSimpleArchive"
+rm -fr "export/$department/Approved"
+rm -fr "export/$department/Multi-Author"
+cd "export/$department"
+unzip DSpaceSimpleArchive.zip
+```
+
+## Multi-Author Submissions
+
+```bash
+/bin/tcsh
+set department="Mechanical & Aerospace Engr"
+source clean-simple-archives
+source init-simple-archives
+```
+
+```bash
+/bin/tcsh
+set department="Mechanical & Aerospace Engr"
+pipenv install lxml pandas
+
+source prepare-to-dataspace "export/$department"
+```
+
+### Transfer the SIPs to the server
```

Comment:
Could you make this H2 instead of H3? I skipped over it when reading because I thought it was under "Multi-Author Submissions"